source/unit/Constraints.ooc

```diff
@@ -131,6 +139,10 @@ EqualModifier: class extends Modifier {
 		f := func (value, c: Object) -> Bool { value as Cell<Double> get() == c as Cell<Double> get() }
 		CompareWithinConstraint new(this, Cell<Double> new(correct), f)
 	}
+	to: func ~ldouble (correct: LDouble) -> CompareWithinConstraint {
+		f := func (value, c: Object) -> Bool { value as Cell<LDouble> get() == c as Cell<LDouble> get() }
+		CompareWithinConstraint new(this, Cell<LDouble> new(correct), f)
+	}
```

Comment:
That's true, there's no point in spending more time on cleaning this up at the moment since the whole thing is getting thrown out soon. I welcome the fixes in this PR, though!